editor/src/components/canvas/canvas-strategies/strategies/convert-to-absolute-and-move-strategy.tsx

```diff
@@ -91,10 +98,13 @@ export function convertToAbsoluteAndMoveStrategy(
     return null
   }
 
-  const target = filteredSelectedElements[0]
-  const autoLayoutSiblings = getAutoLayoutSiblings(canvasState.startingMetadata, target)
+  const autoLayoutSiblings = getAutoLayoutSiblings(canvasState.startingMetadata, originalTargets[0]) // TODO next PR fix for selection length > 1 !!!!!
```

Comment:
we use the original selected elements as the target to determine the siblings bounds. this is in line with @bkrmendy 's fixes from yesterday and today for other strategies.